addons/web/static/src/model/relational_model/utils.js

```diff
@@ -51,8 +52,14 @@ export function makeActiveField({
 const AGGREGATABLE_FIELD_TYPES = ["float", "integer", "monetary"]; // types that can be aggregated in grouped views
 
 export class FetchRecordError extends Error {
```

Comment:
To be consistent, I will put the definition of the error with the handler on his own file (`fetch_record_error.js` or just `errors.js` to be more generic, something like that).